### PR TITLE
Add: percentage event volume 

### PR DESCRIPTION
**Pull request by MimiDumpling**
Added the percentage event volume used to trigger a quota notification.

### CODE REVIEW

**Comment by adhiraj** [comment]
@MimiDumpling I'm assuming we've verified this? if not, perhaps @maheskett can help confirm?

**Review by adhiraj — APPROVED** [review]

**Comment by MimiDumpling** [comment]
> @MimiDumpling I'm assuming we've verified this? if not, perhaps @maheskett can help confirm?

@georgekallilas mentioned the 80% volume threshold. Happy to change it, if necessary.

**Comment by maheskett** [comment]
I added you both to a notion doc around overage notifications. There is an email at 80% usage for errors only, and an email for exceeding quota for each data type. I don't think we send an email when on-demand is activated, but instead at different on-demand thresholds.

**Comment by MimiDumpling** [comment]
> I added you both to a notion doc around overage notifications. There is an email at 80% usage for errors only, and an email for exceeding quota for each data type. I don't think we send an email when on-demand is activated, but instead at different on-demand thresholds.

Thanks, Megan! I'll use the information in the Notion doc to update the PR.

File: src/docs/product/alerts-notifications/notifications/index.mdx [hunk 1]
```diff
@@ -50,7 +50,10 @@ Deploy emails are sent to users who have committed to the release that was deplo
 
 Sentry sends all owners of the organization emails when:
 
-- The event volume is approaching or has exceeded the quota
+- 80% of the error volume has depleted
+- Any data type (errors or events) has exceeded the quota
```
Comment (MimiDumpling) [inline src/docs/product/alerts-notifications/notifications/index.mdx:54]: @maheskett does this include transaction data?

Comment (maheskett) [reply src/docs/product/alerts-notifications/notifications/index.mdx:54]: Yes, I think this sounds reasonable:

```
- 80% of the error volume has been depleted
- Errors or transactions have exceeded quota
```

Also, the notion doc is for our own use as many of these notifications are in flux, so I'd just use your best judgement about what makes sense to share in the public docs.

Comment (MimiDumpling) [reply src/docs/product/alerts-notifications/notifications/index.mdx]: @adhiraj are you okay with these recent updates?

Comment (adhiraj) [reply src/docs/product/alerts-notifications/notifications/index.mdx]: +1 to megan's suggestion "Errors or transactions have exceeded their quota"

**Review by georgekallilas — APPROVED** [review]
Look goods to me!

**Comment by maheskett** [comment]
I'll get this merged once master is passing

File: src/docs/product/alerts-notifications/notifications/index.mdx [hunk 2]
```diff
@@ -50,7 +50,10 @@ Deploy emails are sent to users who have committed to the release that was deplo
 
 Sentry sends all owners of the organization emails when:
 
-- The event volume is approaching or has exceeded the quota
+- 80% of the error volume has depleted
+- Errors or transactions have exceeded the quota
+  - Includes on-demand
+  - Email also sent to billing roles
```
Comment (PeloWriter) [inline src/docs/product/alerts-notifications/notifications/index.mdx:56]: This is a bit confusing as worded. Suggestions below:
```suggestion
- 80% of the organization's error volume has been depleted
- Errors or transactions have exceeded the organization's quota, which includes on-demand capacity
- On-demand capacity is activated
- Spike protection is either activated or deactivated

An email is also sent to the user assigned the billing role for the organization.
```

File: src/docs/product/alerts-notifications/notifications/index.mdx [hunk 3]
```diff
@@ -50,7 +50,10 @@ Deploy emails are sent to users who have committed to the release that was deplo
...
 - On-demand is activated
 - Spike protection is activated or deactivated
```
Comment (PeloWriter) [inline src/docs/product/alerts-notifications/notifications/index.mdx:58]: Incorporated above, so can be removed.
```suggestion

```

**Review by PeloWriter — APPROVED** [review]

File: src/docs/product/alerts-notifications/notifications/index.mdx (hunk 3 above)
Comment (PeloWriter) [reply src/docs/product/alerts-notifications/notifications/index.mdx:58]: Removing these altogether, as this isn't correct.

**Pull request merged**